pytest_fixtures/component/puppet.py

```diff
@@ -152,3 +153,31 @@ def session_puppet_default_os(session_puppet_enabled_sat):
         .search(query={'search': search_string})[0]
         .read()
     )
+
+
+@pytest.fixture(scope='module')
+def module_puppet_published_cv(session_puppet_enabled_sat, module_puppet_org):
```

Comment:
I would move this to CV fixtures instead of keeping it here as other Component owners willing to access puppet sat CV can easily see its availability.

We should move the fixtures based on its return entity. Consider this as a non blocking comment though.